src/components/frame/v5/pages/StreamingPaymentsPage/partials/StreamingPaymentsTable/types.ts

```diff
@@ -0,0 +1,24 @@
+import { type StreamingPayment } from '~types/graphql.ts';
+
+import { type UserStreamsItem } from './partials/UserStreams/UserStreams.ts';
+
+export interface StreamingTableFieldModel {
+  user: string;
+  userStreams: UserStreamsItem[];
```

Comment:
being nit-picky, however as I read the code I understand these are monthly totals for each token, so how about `tokenTotalsPerMonth` or similar?

Reply:
@jakubcolony This is being addressed in - https://github.com/JoinColony/colonyCDapp/pull/3379